loggingservice/requirements.txt

```diff
@@ -1,7 +1,7 @@
 Flask==1.1.1
```

Comment:
```suggestion
Flask==1.1.1
itsdangerous==2.0.1
Jinja2==3.0.3
```
Please include these to fix the build error due to a recent upstream change.

Reply:
@ywkim312, I think this PR can be merged after fixing the build error. Please look into this when you get a chance. Thanks.

Reply:
added